src/proximity-manager/index.ts

```diff
@@ -37,6 +37,11 @@ const IoReactNativeProximity = NativeModules.IoReactNativeProximity
 const BleManagerModule = NativeModules.BleManager;
 const bleManagerEmitter = new NativeEventEmitter(BleManagerModule);
 
+enum ProximityFlowPhases {
```

Comment:
@grausof what do you think to add a description related to the use of this enum?

Reply:
what do you think about:` This enum is used in order to understand what state our proximity flow is in. This is because communication with BLE is asynchronous and therefore not having a request/response mapping it is necessary to keep track of the status to understand the generated response to which request (part of the flow) it refers to.`

Reply:
LGTM